src/test/java/org/opensearch/security/dlic/dlsfls/DlsFlsCrossClusterMinimalRoundtripSearchTests.java

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
I see a few new test files have been added in the PR. Why are they needed?

Reply:
I had to remove use of the Parameterized test runner since Randomized runner was in use, these new files extend the existing tests to keep the test coverage intact with minimal additional code.